### DISCUSSION TITLE
Beatstep Vs  Xtouch Mini

### DISCUSSION THREAD

**Discussion by robreyez** [rsjaffe/MIDI2LR#764] [General]
hello everyone . just wanted to know if there any users out there that have experience with the beatstep and xtouch mini. planning on getting one so any input on build quality, midi2lr software support...etc. thatwould be great, thanks ahead of time.

**Comment by antzpantz** [comment]
Plenty of uses of MIDI2LR use it with the X-Touch Mini, including myself.

**Comment by spacewhippet** [comment]
I use an x-touch mini for my work (very high volume action photography) and it works amazingly with midi2lr.
